src/main/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:17-jdk-slim-bullseye
+FROM eclipse-temurin:17-jre
```

Comment:
We should maybe streamline this over the whole Artemis project? (Thinking about the Github actions and my docker compose refactoring) Maybe in a seperate PR if there are problems to easier revert it? I can do this if desired.

In the bamboo tests (client-tests and server-tests) we are currently using the following docker image as base: https://github.com/ls1intum/docker-jdk-node-yarn which is based on `eclipse-temurin` itself therefore this change here seems really good!

Reply:
Sure, I reverted the change here and will open a separate PR for it.